src/components/StockDetails/StockNews.jsx

```diff
@@ -24,17 +35,32 @@ const StockNews = ({ ...otherProps }) => {
             {isStockNewsLoading ? (
                 <LoadingSpinner />
             ) : (
-                <VStack>
-                    {stockNews.map(({ _id, url, title }) => (
-                        <Box key={_id} w="80%">
-                            <NewsArticle
-                                date="2021-01-01"
-                                title={title}
-                                url={url}
-                            />
-                            <Divider my={4} orientation="horizontal" />
```

Comment:
i think we should keep the divider between the articles
especially important when the window is small since the text bunches up and it's hard to see what text is part of what image

Reply:
@weiyoung and I had a discussion about it 
Can change it to what the concensus is